### DISCUSSION TITLE
Inventree replaces nginx default files on update

### DISCUSSION THREAD

**Discussion by rohitsingal1** [inventree/InvenTree#5202] [General]
Whenever inventree gets updated it replaces the default files inside sites-enabled folder with inventree nginx configuration.
Although I keep a backup of all my configuration files but still its a hindrance that the files have to be replaced after every update

**Comment by SchrodingersGat** [comment]
@rohitsingal1 I assume you are using the package installer?

**Comment by rohitsingal1** [comment]
Yes. After that ubuntu runs updates automatically.

**Comment by rieders** [comment]
Hallo
Ich habe auch ein Plugin installiert und danach mein Server neu gestartet.
Nun erscheint nur die Apache2 Debian Default Page. 
Wie komme ich wieder auf die Login Page?
Grüße
